RetailCoder.VBE/Properties/Settings.settings

```diff
@@ -0,0 +1,149 @@
+﻿<?xml version='1.0' encoding='utf-8'?>
+<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)" GeneratedClassNamespace="Rubberduck.Properties" GeneratedClassName="Settings">
+  <Profiles />
+  <Settings>
+    <Setting Name="DefaultHotkey_CodePaneRefactorRenameCommand" Type="Rubberduck.Settings.HotkeySetting" Scope="Application">
+      <Value Profile="(Default)">&lt;?xml version="1.0" encoding="utf-16"?&gt;
+&lt;HotkeySetting xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"&gt;
```

Comment:
I do not know how to feel about this - I really don't like xml embedded in another xml. Is there no other way to make this more human-readable? Surely the settings XML will allow you to create custom elements?

Reply:
The `Settings` file is an XML file. You can serialize your class in a couple of ways: binary, string, XML or by writing a custom provider. I decided that XML is the best option.

What I think is more important here is that I don't see this behaviour. When I open the `.settings` file in Visual Studio, the `Value` column contains proper XML...

Reply:
Now that I think of it, probably this kind of unreadable gibberish appear only on Github.

Reply:
Oh, totally. My concern was more that it would be difficult to do code review when editing the settings from GitHub if we had to read that way. That's why I prefaced my comment with not knowing how to feel about that. It's just that I like having everything easily accessible without having to first deserialize it into some other format. *cough*Lazy*cough*. :)